src/Microsoft.ML.OnnxConverter/OnnxContextImpl.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.ML.Model.OnnxConverter
     /// </summary>
     internal sealed class OnnxContextImpl : OnnxContext
     {
+        public const int CurrentOpSetVersion = 11;
```

Comment:
>11 [](start = 47, length = 2)

Isn't it 12 now?

Reply:
@harishsk Do we want to upgrade the current Opset version to 12 soon, I can do that in this PR

Reply:
Yes, Please upgrade.

---
In reply to: [433383458](https://github.com/dotnet/machinelearning/pull/5176#discussion_r433383458) [](ancestors = 433383458)